hoomd/md/dihedral.py

```diff
@@ -119,10 +118,13 @@ def __init__(self):
         self._add_typeparam(params)
 
 
-def _table_eval(theta, V, T, width):
-    dth = (2 * math.pi) / float(width - 1)
-    i = int(round((theta + math.pi) / dth))
-    return (V[i], T[i])
+class Harmonic(Periodic):
+    """Periodic dihedral force.
+
+    .. deprecated:: v3.4.0
```

Comment:
Shouldn't the deprecation version match the version of the next release?

Reply:
Yes, it should match the next release.